CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Fixed
 Added
 -----
 - Added support for probabilities snapshot and Pauli expectation value snapshot in the stabilizer simulator (\#423)
+- MPS simulation method: added support for ``mcx`` (\#454)
```

Comment:
`mcx` is a multi-controlled-X gate with an arbitrary number of control qubits. I think in this case you have only added support for a the Toffoli gate, so it should just say `ccx` here.

```suggestion
- MPS simulation method: added support for ``ccx`` (\#454)
```

Reply:
done